Resync `mathml` from WPT Upstream

Ahmad-S792 opened the pull request:
#### 258739f6c5c15d35e573a5dea93cd3dc36fe3fbd
<pre>
Resync `mathml` from WPT Upstream
<a href="https://bugs.webkit.org/show_bug.cgi?id=297002">https://bugs.webkit.org/show_bug.cgi?id=297002</a>
<a href="https://rdar.apple.com/157670185">rdar://157670185</a>

Reviewed by Tim Nguyen.

Upstream commit: <a href="https://github.com/web-platform-tests/wpt/commit/f6caca9f8f55eef2aee45858219bcd876485e62a">https://github.com/web-platform-tests/wpt/commit/f6caca9f8f55eef2aee45858219bcd876485e62a</a>

* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/mpadded/mpadded-legacy-length-values-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/mpadded/mpadded-legacy-length-values.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/mrow/dynamic-mrow-like-001-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/mrow/dynamic-mrow-like-001-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/mrow/dynamic-mrow-like-001.html:
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/mrow/inferred-mrow-baseline.html:
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/mrow/inferred-mrow-stretchy-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/mrow/inferred-mrow-stretchy.html:
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/mrow/legacy-mrow-like-elements-001-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/mrow/legacy-mrow-like-elements-001.html:
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/mrow/merror-001.html:
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/mrow/mphantom-001.html:
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/mrow/mrow-preferred-width.html:
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/mrow/spacing-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/mrow/spacing.html:
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/operators/character-level-mirroring-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/operators/character-level-mirroring.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/operators/embellished-operator-001-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/operators/embellished-operator-001.html:
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/operators/mo-lspace-rspace-legacy-length-values-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/operators/mo-lspace-rspace-legacy-length-values.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/operators/mo-minsize-maxsize-legacy-length-values-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/operators/mo-minsize-maxsize-legacy-length-values.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/operators/symmetric-largeop-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/operators/symmetric-largeop.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/operators/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/radicals/glyph-level-mirroring-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/radicals/glyph-level-mirroring.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/radicals/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/spaces/mspace-legacy-length-values-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/spaces/mspace-legacy-length-values.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/spaces/space-like-003.html:
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/tables/table-default-styles-001.html:
* LayoutTests/imported/w3c/web-platform-tests/mathml/relations/css-styling/lengths-1-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/mathml/relations/css-styling/lengths-1-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/mathml/relations/css-styling/lengths-1.html:
* LayoutTests/imported/w3c/web-platform-tests/mathml/relations/css-styling/lengths-2-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/mathml/relations/css-styling/lengths-2.html:
* LayoutTests/imported/w3c/web-platform-tests/mathml/relations/css-styling/lengths-vi-vb-units-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/mathml/relations/css-styling/lengths-vi-vb-units.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/mathml/relations/css-styling/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/mathml/relations/css-styling/width-height-001-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/mathml/relations/css-styling/width-height-001.html:
* LayoutTests/imported/w3c/web-platform-tests/mathml/tools/css-units.py: Added.
* LayoutTests/imported/w3c/web-platform-tests/mathml/tools/largeop.py:
* LayoutTests/imported/w3c/web-platform-tests/mathml/tools/radicals.py:
(createStretchyRadical):
* LayoutTests/imported/w3c/web-platform-tests/mathml/tools/stretchy.py:
* LayoutTests/imported/w3c/web-platform-tests/mathml/tools/w3c-import.log:
* LayoutTests/platform/ios/TestExpectations:
* LayoutTests/platform/mac/imported/w3c/web-platform-tests/mathml/presentation-markup/mrow/inferred-mrow-baseline-expected.txt:
* LayoutTests/platform/mac/imported/w3c/web-platform-tests/mathml/presentation-markup/mrow/mrow-preferred-width-expected.txt:
* LayoutTests/tests-options.json:
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/mathml/presentation-markup/mrow/inferred-mrow-baseline-expected.txt:
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/mathml/presentation-markup/mrow/mrow-preferred-width-expected.txt:
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/mathml/presentation-markup/operators/character-level-mirroring-expected.txt:
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/mathml/presentation-markup/radicals/glyph-level-mirroring-expected.txt:
* LayoutTests/platform/glib/TestExpectations:
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/operators/character-level-mirroring-expected.txt:
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/mathml/presentation-markup/mrow/inferred-mrow-baseline-expected.txt:
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/mathml/presentation-markup/mrow/mrow-preferred-width-expected.txt:
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/mathml/relations/css-styling/lengths-2-expected.txt:
* LayoutTests/TestExpectations:

Canonical link: <a href="https://commits.webkit.org/298510@main">https://commits.webkit.org/298510@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/9a9dd059547215abc52e27a2f7743cdde79d0afa

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows | Apple Internal |
| ----- | ---------------------- | ------- |  ----- |  --------- | ------ |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/115649 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/35312 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/25835 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/121696 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/66163 "Built successfully") | [✅ 🛠 ios-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/560fdc4b-8654-4e5f-bd94-cb6e5bef63b7) 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/117538 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/35993 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/43895 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/87850 "Passed tests") | [💥 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/42480 "An unexpected error occured. Recent messages:Printed configuration") | [✅ 🛠 mac-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/385de054-5287-4dcd-83de-719637214033) 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/118597 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/28714 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/103789 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/68250 "Passed tests") | | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/27851 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/21905 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/65373 "Built successfully") | | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/98094 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/22026 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/124845 "Built successfully") | | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/42544 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/31898 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/96605 "Passed tests") | | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/42911 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/99979 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/96393 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/24541 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/41649 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/19508 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/38422 "Built successfully") | | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/42435 "Built successfully") | | [⏳ 🛠 jsc-armv7 ](https://ews-build.webkit.org/#/builders/JSC-ARMv7-32bits-Build-EWS "Waiting in queue, processing has not started yet") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/41908 "Built successfully") | | [⏳ 🧪 jsc-armv7-tests ](https://ews-build.webkit.org/#/builders/JSC-ARMv7-32bits-Build-EWS "Waiting in queue, processing has not started yet") | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/45239 "Built successfully") | | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/43616 "Built successfully") | | | | 
<!--EWS-Status-Bubble-End-->

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/da635354348e401e7375405d84310d42c1b94643. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/49012

Ahmad-S792 committed:
Resync `mathml` from WPT Upstream
https://bugs.webkit.org/show_bug.cgi?id=297002
rdar://157670185

Reviewed by Tim Nguyen.

Upstream commit: https://github.com/web-platform-tests/wpt/commit/f6caca9f8f55eef2aee45858219bcd876485e62a

* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/mpadded/mpadded-legacy-length-values-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/mpadded/mpadded-legacy-length-values.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/mrow/dynamic-mrow-like-001-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/mrow/dynamic-mrow-like-001-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/mrow/dynamic-mrow-like-001.html:
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/mrow/inferred-mrow-baseline.html:
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/mrow/inferred-mrow-stretchy-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/mrow/inferred-mrow-stretchy.html:
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/mrow/legacy-mrow-like-elements-001-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/mrow/legacy-mrow-like-elements-001.html:
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/mrow/merror-001.html:
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/mrow/mphantom-001.html:
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/mrow/mrow-preferred-width.html:
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/mrow/spacing-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/mrow/spacing.html:
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/operators/character-level-mirroring-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/operators/character-level-mirroring.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/operators/embellished-operator-001-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/operators/embellished-operator-001.html:
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/operators/mo-lspace-rspace-legacy-length-values-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/operators/mo-lspace-rspace-legacy-length-values.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/operators/mo-minsize-maxsize-legacy-length-values-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/operators/mo-minsize-maxsize-legacy-length-values.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/operators/symmetric-largeop-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/operators/symmetric-largeop.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/operators/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/radicals/glyph-level-mirroring-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/radicals/glyph-level-mirroring.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/radicals/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/spaces/mspace-legacy-length-values-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/spaces/mspace-legacy-length-values.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/spaces/space-like-003.html:
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/tables/table-default-styles-001.html:
* LayoutTests/imported/w3c/web-platform-tests/mathml/relations/css-styling/lengths-1-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/mathml/relations/css-styling/lengths-1-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/mathml/relations/css-styling/lengths-1.html:
* LayoutTests/imported/w3c/web-platform-tests/mathml/relations/css-styling/lengths-2-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/mathml/relations/css-styling/lengths-2.html:
* LayoutTests/imported/w3c/web-platform-tests/mathml/relations/css-styling/lengths-vi-vb-units-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/mathml/relations/css-styling/lengths-vi-vb-units.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/mathml/relations/css-styling/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/mathml/relations/css-styling/width-height-001-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/mathml/relations/css-styling/width-height-001.html:
* LayoutTests/imported/w3c/web-platform-tests/mathml/tools/css-units.py: Added.
* LayoutTests/imported/w3c/web-platform-tests/mathml/tools/largeop.py:
* LayoutTests/imported/w3c/web-platform-tests/mathml/tools/radicals.py:
(createStretchyRadical):
* LayoutTests/imported/w3c/web-platform-tests/mathml/tools/stretchy.py:
* LayoutTests/imported/w3c/web-platform-tests/mathml/tools/w3c-import.log:
* LayoutTests/platform/ios/TestExpectations:
* LayoutTests/platform/mac/imported/w3c/web-platform-tests/mathml/presentation-markup/mrow/inferred-mrow-baseline-expected.txt:
* LayoutTests/platform/mac/imported/w3c/web-platform-tests/mathml/presentation-markup/mrow/mrow-preferred-width-expected.txt:
* LayoutTests/tests-options.json:
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/mathml/presentation-markup/mrow/inferred-mrow-baseline-expected.txt:
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/mathml/presentation-markup/mrow/mrow-preferred-width-expected.txt:
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/mathml/presentation-markup/operators/character-level-mirroring-expected.txt:
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/mathml/presentation-markup/radicals/glyph-level-mirroring-expected.txt:
* LayoutTests/platform/glib/TestExpectations:
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/operators/character-level-mirroring-expected.txt:
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/mathml/presentation-markup/mrow/inferred-mrow-baseline-expected.txt:
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/mathml/presentation-markup/mrow/mrow-preferred-width-expected.txt:
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/mathml/relations/css-styling/lengths-2-expected.txt:
* LayoutTests/TestExpectations:

Canonical link: https://commits.webkit.org/298510@main

webkit-commit-queue commented:
Committed 298510@main (258739f6c5c1): <https://commits.webkit.org/298510@main>

Reviewed commits have been landed. Closing PR #49012 and removing active labels.
